### ISSUE TITLE
Mac OS: cca_functions.py _load_files error and quick fix

### ISSUE THREAD

**Issue by jordanyx** [SchmollerLab/Cell_ACDC#24]
**Describe the bug**
The _load_files function within cca_functions.py (used in the jupyter notebook cell_cycle_analysis.ipynb) fails when used on MacOS; I'm under the impression it's a file name appending issue, but not certain.

**Apparent solution**
Within the cca_functions.py file, we manually changed all 15 instances of "\*" to "/*". After that, the jupyter notebook was able to run normally. I'm not certain why this worked, but I believe it has to do with different file path syntax between Windows and Unix (forward vs. back slash.

Just putting this on your radar; not sure what the best general solution is.

**Comment by ElpadoCan** [comment]
Yes you are right, the `\` works only for Windows path, while for UNIX path you need `/`. We completely forgot that. Your fix works fine, but it would not work on Windows. 

The way to fix it to be OS-indipendent is to use the python module called `os` and join paths with `os.path.join` which takes care of the separator depending on the OS. 

 I fixed it in the latest release `v1.2.4rc12`, update with `pip install --upgrade cellacdc`, thanks!
